Gujarati/ઉત્તરાયણ/ઉત્તરાયણમાં ખેંચ કે ઢીલ.md

```diff
@@ -0,0 +1,5 @@
+ઉત્તરાયણનો સુંદર દિવસ છે. વાયરો વંટોળની જેમ વાયી રહ્યો છે. પતંગને ખાલી પકડી રાખવો પણ ભારે પડી રહ્યો છે. અાજુ બાજુ પતંગોથી આસમાન ભરાઇ ગયું છે. બધા પતંગ કાપવા તમારી લાળ પડી રહી છે.
+તમે કયી રીત અપનાવશો?
+
+- [ઢીલ](ખેંચ/ખેંચ ખેંચ ખેંચ ખેંચ... એ કાપ્યો છે.md)
+- [ખેંચ](ઢીલ/ઢીલ દે ઢીલ દે દે રે ભઇયા.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.